label_studio_converter/converter.py

```diff
@@ -524,10 +524,7 @@ def add_image(images, width, height, image_id, image_path):
                 annotation_id = len(annotations)
 
                 if 'rectanglelabels' in label or 'labels' in label:
-                    x = int(label['x'] / 100 * width)
-                    y = int(label['y'] / 100 * height)
-                    w = int(label['width'] / 100 * width)
-                    h = int(label['height'] / 100 * height)
+                    x, y, w, h = self.rotated_rectangle(label)
```

Comment:
`self.rotated_rectangles()` is returning values in percentage scale and it should be in pixel scale.